docs/pages/cloud/faq.mdx

```diff
@@ -70,18 +79,18 @@ After [connecting](#how-can-i-access-the-tctl-admin-tool) `tctl` to Teleport Clo
 $ tctl nodes add --ttl=5m --roles=node,proxy --token=$(uuid)
 ```
 
-## When will a security audit be available?
+## Is an independent security audit available?
 
-Security audit has been completed and is available [here](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/).
+A security audit has been completed and is available [here](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/).
```

Comment:
```suggestion
A security audit has been completed and is available at our [Audit Report 2021](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/) page.
```
Added a suggestion to make this a bit clearer (https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/#here-conceals-what-users-are-clicking).